pkg/externalresource/external_resource.go

```diff
@@ -33,9 +33,9 @@ type Section map[string]*ExternalResource
 
 // ExternalResource represents information on an external resource
 type ExternalResource struct {
-	Icon      string
-	Notes     *Notes
-	Endpoints []*ExternalEndpoint
+	Icon      string              `json:"icon,omitempty" yaml:"icon,omitempty"`
```

Comment:
why i this necesary? i think the exported model is declared at the serializer ? https://github.com/okteto/okteto/blob/3f323291cb3dc189175c92cf38d6299a25ab2721/pkg/externalresource/serializer.go#L22

Reply:
as all the other structs of the manifest, they are annotated for the marshallers/unmarshallers using the yaml/json annotations, as they were not annotated, they were ingored by GetStructKeys